test/hotspot/jtreg/runtime/Nestmates/protectionDomain/TestDifferentProtectionDomains.java

```diff
@@ -93,7 +76,8 @@
                 // Check for target class
                 if (name.startsWith(TARGET)) {
                     try {
-                        byte[] buff = getBytesForClass(name);
+                        String clzFile = name.replaceAll("\\.", "/") + ".class";
+                        byte[] buff = getResourceAsStream(clzFile).readAllBytes();
```

Comment:
There's a ClassUnloadCommon.getBytesForClass(name) that you could have called that does essentially what these two lines do.  Maybe it's different enough to have this, so it's up to you to change it.